crates/theme/src/schema.rs

```diff
@@ -1309,11 +1313,17 @@ pub struct HighlightStyleContent {
 
     #[serde(deserialize_with = "treat_error_as_none")]
     pub font_weight: Option<FontWeightContent>,
+
+    #[serde(deserialize_with = "treat_error_as_none")]
+    pub background_color: Option<String>,
```

Comment:
Very minor nit, but can we put the `background_color` field below the `color` field (both here in the `struct` decl, but also at the initialization sites)?

Reply:
I can do, the reason I did the order this way is defined in `HighlightStyle`

https://github.com/zed-industries/zed/blob/fcb4abf18b20281502b48e8691beede2b37934d8/crates/gpui/src/style.rs#L297-L317

Reply:
I've just realised the `font_weight` and `font_style` fields are flipped so I guess my reasoning doesn't really hold 😄

Reply:
@maxdeviant I've changed the order